[7.10][DOCS] Adds ML related release highlights to What's new. (#63934)
elastic/elasticsearch: docs/reference/release-notes/highlights.asciidoc

the `data_content` tier automatically. Nodes with the pre-existing `data` role are 
considered to be part of all tiers.

[discrete]
[[auc-roc-eval-class]]
=== AUC ROC evaluation metrics for classification analysis

{ml-docs}/ml-dfanalytics-evaluate.html#ml-dfanalytics-class-aucroc[Area under the curve of receiver operating characteristic (AUC ROC)] 
is an evaluation metric that has been available for {oldetection} since 7.3 and 

> now is available for {classification} analysis as well. It represents the 

Reviewer: ```suggestion
now is available for {classification} analysis. AUC ROC represents the 
```

performance of the {classification} process at different predicted probability 
thresholds. The true positive rate for a specific class is compared against the 
rate of all the other classes combined at the different threshold levels to 
create the curve.

[discrete]
[[custom-feature-processor-dfa]]
=== Custom feature processors in {dfanalytics}

Feature processors enable you to extract process features from document fields. 
You can use these features in model training and model deployment. Custom 
feature processors provide a mechanism to create features that can be used at 

> search and ingest time and they don’t take up space in the index. Furthermore, 
> this process more tightly couples feature generation with the resulting model. 

Reviewer: ```suggestion
search and ingest time and they don’t take up space in the index.
This process more tightly couples feature generation with the resulting model. 
```
